app/workers/github_repos/repo_sync_worker.rb

```diff
@@ -0,0 +1,35 @@
+module GithubRepos
+  class RepoSyncWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 10
+
+    def perform(repo_id)
+      repo = GithubRepo.find_by(id: repo_id)
+      return unless repo
+
+      user = repo.user
+      return unless user.identities.github.exists?
```

Comment:
Added since we have cases where users with GithubRepos don't have Github identities.